plugin/Filter/DefaultFilterBuilderSimilarEstates.php

```diff
@@ -84,17 +84,18 @@ public function buildFilter(): array
 			$filter['plz'] []= ['op' => '=', 'val' => $pFilterConfiguration->getPostalCode()];
 		}
 
-        if ($pDataListView->getDontShowArchived() && $showArchived !== 'status2obj_archiviert') {
-            $filter['status2'] []= ['op' => '!=', 'val' => ['status2obj_archiviert'] ];
-        }
+		if ($pDataListView->getShowArchived() && $showArchived !== 'status2obj_archiviert') {
+			$filter['status2'] [] = ['op' => '!=', 'val' => ['status2obj_archiviert']];
```

Comment:
The getter got inverted, but the boolean logic and the filter weren't. I guess that's not on purpose?

`status2` is still an issue.